src/prefect/input/run_input.py

```diff
@@ -1,3 +1,64 @@
+"""
+This module offers a framework for data exchange between flow runs enabling
```

Comment:
I would combine these paragraphs and stick to what users can do with it:

```suggestion
This module contains functions that allow sending type-checked `RunInput` data
to flows at runtime. Flows can send back responses, establishing two-way
channels with senders. These functions are particularly useful for systems that
require ongoing data transfer or need to react to input quickly.
```